### PR TITLE
[GeoMechanicsApplication] Remove the "solution_type" = "K0-procedure"

### LINKED ISSUE #12694: [GeoMechanicsApplication] Remove the "solution_type" = "K0-procedure"
Replace the "solution_type" = "K0-procedure" by "quasi_static" in the project parameters .json files and remove the possibility from geomechanics_UPw_solver.py.

**Background**
This solution type appears in 12 tests and the UPw solver

### PR DESCRIPTION
**Pull request by markelov208**
**📝 Description**
A brief description of the PR.

- Replaced the "solution_type" = "K0-procedure" by "quasi_static" in the project parameters .json files
- Removeed the possibility from geomechanics_UPw_solver.py.

### CODE REVIEW

File: applications/GeoMechanicsApplication/python_scripts/geomechanics_U_Pw_solver.py [hunk 1]
```diff
@@ -187,13 +187,6 @@ def _ConstructScheme(self, scheme_type, solution_type):
             elif (solution_type.lower() == "dynamic"):
                 KratosMultiphysics.Logger.PrintInfo("GeoMechanics_U_Pw_Solver, scheme", "Dynamic.")
                 scheme = KratosGeo.NewmarkDynamicUPwScheme(beta,gamma,theta)
-            elif (solution_type.lower() == "k0-procedure" or solution_type.lower() == "k0_procedure"):
-                if (rayleigh_m < 1.0e-20 and rayleigh_k < 1.0e-20):
-                    KratosMultiphysics.Logger.PrintInfo("GeoMechanics_U_Pw_Solver, scheme", "Quasi-UnDamped.")
-                    scheme = KratosGeo.NewmarkQuasistaticUPwScheme(beta,gamma,theta)
-                else:
-                    KratosMultiphysics.Logger.PrintInfo("GeoMechanics_U_Pw_Solver, scheme", "Quasi-Damped.")
-                    scheme = KratosGeo.NewmarkQuasistaticDampedUPwScheme(beta,gamma,theta)
```
Comment (rfaasse) [inline applications/GeoMechanicsApplication/python_scripts/geomechanics_U_Pw_solver.py:196]: There is another k0-procedure in line 196 (for the other scheme), I'd suggest to remove that one as well

**Review by rfaasse — COMMENTED** [review]
Thanks for this clean-up, there seems to be one more branch in the solver where it checks for 'k0-procedure', other than that I think it's ready to go!

**Review by WPK4FEM — APPROVED** [review]
Looks consistent. I think these were all appearances of the solution_type K0_procedure.
Good to go.

File: applications/GeoMechanicsApplication/python_scripts/geomechanics_U_Pw_solver.py (hunk 1 above)
Comment (markelov208) [reply applications/GeoMechanicsApplication/python_scripts/geomechanics_U_Pw_solver.py:196]: Richard, thank you very much. It has been deleted.

**Review by rfaasse — APPROVED** [review]
Thanks for incorporating the changes, this looks good to go!

**Pull request merged**